### ISSUE TITLE
Hanging at Parsing MSA

### ISSUE THREAD

**Issue by kevinmyers** [amkozlov/raxml-ng#199]
We are attempting to make a tree using RAxML-ng adaptive (v1.2.1-adaptive). The alignment file (attached in .txt form for GitHub) was created using `jackhmmer` in Stockholm format and then converted to aligned fasta format:

`jackhmmer -A mDHFR.sto -o mDHFR.out.txt mDHFR.fasta uniref90.fasta`
`/hmmer-3.1b2-linux-intel-x86_64/binaries/esl-reformat -u -o C_tails_full_seqs.fasta afa mDHFR.sto`

When we run `raxml-ng-adaptive` in the `--parse` option, it hangs at the `Predicting difficulty of the MSA . . . ` step:

```
/opt/bifxapps/raxml-ng-adaptive/bin/raxml-ng-adaptive --parse --msa C_tails_full_seqs.fasta -model LG+G8+F --prefix T1

RAxML-NG v. 1.2.1-adaptive released on 11.04.2024 by The Exelixis Lab.
Developed by: Alexey M. Kozlov and Alexandros Stamatakis.
Contributors: Diego Darriba, Tomas Flouri, Benoit Morel, Sarah Lutteropp, Ben Bettisworth, Julia Haag, Anastasis Togkousidis.
Latest version: https://github.com/amkozlov/raxml-ng
Questions/problems/suggestions? Please visit: https://groups.google.com/forum/#!forum/raxml

System: Intel(R) Xeon(R) CPU E5-2683 v4 @ 2.10GHz, 32 cores, 251 GB RAM

RAxML-NG was called at 15-Oct-2024 09:24:41 as follows:

/opt/bifxapps/raxml-ng-adaptive/bin/raxml-ng-adaptive --parse --msa C_tails_full_seqs.fasta -model LG+G8+F --prefix T1

Analysis options:
  run mode: Alignment parsing and compression
  start tree(s): 
  random seed: 1729002281
  tip-inner: OFF
  pattern compression: ON
  per-rate scalers: OFF
  site repeats: ON
  logLH epsilon: general: 0.100000, brlen-triplet: 1000.000000
  branch lengths: proportional (ML estimate, algorithm: NR-FAST)
  SIMD kernels: AVX2
  parallelization: coarse-grained (auto), PTHREADS (auto)

[00:00:00] Reading alignment from file: C_tails_full_seqs.fasta
[00:00:00] Loaded alignment with 1497 taxa and 2550 sites

Alignment comprises 1 partitions and 2403 patterns

Partition 0: noname
Model: LG+FC+G8m
Alignment sites / patterns: 2550 / 2403
Gaps: 87.51 %
Invariant sites: 26.47 %


[00:00:00] Adaptive mode: Predicting difficulty of the MSA ...
```

We have let it run over a weekend and it has shown no progress. We have successfully used raxml-ng-adaptive (and regular raxml-ng) in the past so I'm confused what would be causing this issue. Any help would be greatly appreciated.

[C_tails_full_seqs.fasta.txt](https://github.com/user-attachments/files/17380198/C_tails_full_seqs.fasta.txt)

**Comment by amkozlov** [comment]
Hi @kevinmyers,
could you please upgrade to the latest version and try again?

With `RAxML-NG v. 1.2.1-adaptive released on 02.10.2024` It took ~7min on my laptop.

**Comment by kevinmyers** [comment]
I believe we have a more recent version?

`RAxML-NG v. 1.2.1-adaptive released on 11.04.2024`

**Comment by amkozlov** [comment]
now it's April 11th, 2024 :) 

please pull the most recent version from github:

https://github.com/amkozlov/raxml-ng/wiki/Installation#building-adaptive-branch

**Comment by kevinmyers** [comment]
That's what I get for reading it in the US date format - apologies!

**Comment by kevinmyers** [comment]
We updated to the newest version (`RAxML-NG v. 1.2.2-master released on 30.04.2024`) and we were able to run the parse command (and the main command) no problem. Thanks for your help!
